src/app/components/core/configuration-manager/kv-list-type-configuration/kv-list-type-configuration.component.ts

```diff
@@ -25,6 +25,9 @@ export class KvListTypeConfigurationComponent implements OnInit {
 
   ngOnInit() {
     this.form = this.rootFormGroup.control;
+    if (this.configuration.items == 'enumeration' && this.configuration.options && typeof(this.configuration.options) == 'string'){
+      this.configuration.options = JSON.parse(this.configuration.options)
+    }
```

Comment:
Do we really need this check? To me API always returns options as a JSON array so no need of JSON.parse here. 
If backend is returning options in a string then it's a backed issue and we should log that?